appeals/web/src/server/views/patterns/change-page-full-width.pattern.njk

```diff
@@ -0,0 +1,33 @@
+{%- extends "app/layouts/app.layout.njk" -%}
```

Comment:
This pattern template looks identical to `change-page.pattern.njk` apart from using a different layout. It would be good to avoid the duplication if possible. The duplicated parts could probably be moved into an `include` and imported into `change-page.pattern.njk` and `change-page-full-width.pattern.njk`.